cmd/gitops/install/cmd_test.go

```diff
@@ -0,0 +1,11 @@
+package install
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestPostRunDefined(t *testing.T) {
+	assert.NotNil(t, Cmd.PostRun, "PostRun should be defined for install")
```

Comment:
I assume this was in the old file, but seems like an odd assertion. Any idea what the back story is here?

Reply:
Nope. These two tests were both in the `cmd_test.go` file for the `gitops` command and I split them into `install` and `uninstall` since we don't have that cmd level any more. I wondered about it too. I see that Mark E put it in. I'll ask him about it.

Reply:
Turns out those are in there to ensure that the checkpoint calls from install/uninstall don't get removed.